css/program.css

```diff
@@ -354,10 +356,6 @@ img {
     padding: 4.5rem 6rem !important;
   }
 
-  .programlist-subscribe .form-container .subscribe-inputbox{
-    flex-direction: column;
```

Comment:
Moving away from flex is not ideal. Can you use justify-content, align-self, align-items, or other while staying with flex? Static assignment of 2rem of padding is not as preferable as staying with flex. You can use `space-around` to help with padding while using flex. You can assign padding while using flex, too. But... try to keep using flex. 😉

Reply:
// @UtkarshMishra12 please update.